### PR TITLE
feat: add `enabled_properties` to costQuerySetOption & create `Add More` button

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description

https://github.com/cloudforet-io/console/assets/18563857/cb50fa16-4bf1-4245-a9b8-cab7828e3978

### CODE REVIEW

File: apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisFiltersAddMoreButton.vue [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+onClickOutside(contextMenuRef, hideContextMenu);
+
+const handleClickAddMore = () => {
+    if (visibleMenu.value) {
+        hideContextMenu();
+    } else {
+        initiateMenu();
+        showContextMenu();
+    }
+};
```
Comment (sulmoJ) [inline apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisFiltersAddMoreButton.vue:54]: It seems like this code isn't working.
(I checked it in the preview.)

**Review by sulmoJ — COMMENTED** [review]
I left a comment!

File: apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisFiltersAddMoreButton.vue [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+</script>
+
+<template>
+    <div class="cost-analysis-add-more-button">
+        <p-button ref="targetRef"
+                  style-type="transparent"
+                  icon-left="ic_plus_bold"
+                  @click="handleClickAddMore"
+        >
+            Add More
```
Comment (sulmoJ) [inline apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisFiltersAddMoreButton.vue:86]: Translation!

File: apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisFiltersAddMoreButton.vue (hunk 1 above)
Comment (yuda110) [reply apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisFiltersAddMoreButton.vue:55]: I fixed it!

**Review by sulmoJ — APPROVED** [review]
LGTM!!!!!!

**Pull request merged**